tesseract/tesseract_planning/tesseract_process_managers/src/process_managers/raster_process_manager.cpp

```diff
@@ -118,8 +118,8 @@ bool RasterProcessManager::init(ProcessInput input)
       // robust because planners could modify composite size, which is rare but does happen when using OMPL where it is
       // not possible to simplify the trajectory to the desired number of states.
       ProcessInput transition_input = input[input_idx][transition_step_idx];
-      transition_input.start_instruction_ptr = input[input_idx - 1].results;
-      transition_input.end_instruction_ptr = input[input_idx + 1].results;
+      transition_input.setStartInstruction(input[input_idx - 1].results);
```

Comment:
This should be setStartInstructionPtr if not it will make a copy of what is currently in the results instead of a pointer to the results data structure which gets updated.